cmd/edenStart.go

```diff
@@ -111,7 +109,7 @@ var startCmd = &cobra.Command{
 				log.Infof("EVE is starting in Parallels")
 			}
 		} else if devModel == defaults.DefaultVBoxModel {
-			if err := eden.StartEVEVBox(vmName, eveImageFile, cpus, mem, hostFwd, getUplinkPortIPMap()); err != nil {
```

Comment:
Doesn't it break something. I think we did this getUplinkPortIPMap for the case when we have several adapters

Reply:
We still have several network adapters (two exactly: `eth0` and `eth1`). What changed is how we get DHCP-allocated IP addresses for those adapters. `getUplinkPortIPMap` would get this information from the controller, but this doesn't work for the first `./eden eve start` because device is not yet onboarded at that point. With this PR DHCP information is obtained directly from the Vbox DHCP server instead.

Reply:
what if it's not Vbox

Reply:
For qemu and parallels it is not needed specify guest IP in port forwarding rules.
Do you want me to leave `getUplinkPortIPMap` there even if it is not needed anymore?

Reply:
I'm ok for that